server/src/routers/MenuItemsRouter/MenuItemsRouter.js

```diff
@@ -0,0 +1,22 @@
+import express from 'express';
+import getMenuIdInfo from './getMenuIdInfo';
+
+const router = express.Router();
+export default router;
+
+router.get('/', async (req, res) => {
+    if (!('menuitemid' in req.query)) {
+        console.warn("menuitemid is an essential parameter");
+        res.sendStatus(500);
```

Comment:
I think you can send a 400 here -- that's 'bad request'.

Reply:
Good idea!